building/configlet/README.md

```diff
@@ -4,28 +4,28 @@
 
 ## Linting
 
-The primary function of configlet is to do _linting_: checking if a track's configuration files are properly structured - both syntactically and semantically. Misconfigured tracks may not sync correctly, may look wrong on the website, or may present a suboptimal user experience, so configlet's guards play an important part in maintaining the integrity of Exercism. The full list of rules that are checked by the linter can be found [here](./linting.md).
+The primary function of configlet is to do _linting_: checking if a track's configuration files are properly structured - both syntactically and semantically. Misconfigured tracks may not sync correctly, may look wrong on the website, or may present a suboptimal user experience, so configlet's guards play an important part in maintaining the integrity of Exercism. The full list of rules that are checked by the linter can be found [here](./lint).
```

Comment:
@iHiD So this PR got merged and deployed, and when I click this link on https://exercism.lol/docs/building/configlet, I get a 500 error. I'm guessing it's because it tries to take me to https://exercism.lol/docs/building/lint instead of https://exercism.lol/docs/building/configlet/lint

The same is true about some other links that starts with `./`.

Reply:
I guess @ee7 noticed the same thing at the same time and opened https://github.com/exercism/docs/pull/160 :)

Reply:
I saw this too. I just created https://github.com/exercism/docs/pull/160 to follow up.